interfaces/builtin/openvswitch.go

```diff
@@ -30,12 +30,14 @@ const openvswitchBaseDeclarationSlots = `
 `
 
 // List of sockets we want to allow access to. This list currently includes
-// sockets needed by ovs-vsctl and ovs-ofctl commands. The latter requires
-// access to per-bridge sockets e.g. for bridge br-data you would need access
-// to /run/openvswitch/br-data.mgmt.
+// sockets needed by ovs-vsctl, ovs-ofctl and ovs-appctl commands. ovs-ofctl
+// requires access to per-bridge sockets e.g. for bridge br-data you would need
+// access to /run/openvswitch/br-data.mgmt.
 const openvswitchConnectedPlugAppArmor = `
 /run/openvswitch/db.sock rw,
 /run/openvswitch/*.mgmt rw,
+/run/openvswitch/ovs-vswitchd.*.ctl rw,
+/run/openvswitch/ovs-vswitchd.pid rw,
```

Comment:
I see from the bug that this is opened O_RDWR, but for this action, is write actually needed?

That said, the man page for `ovs-appctl` (http://www.openvswitch.org/support/dist-docs/ovs-appctl.8.txt) says:
```
       Open vSwitch daemons accept certain  commands  at  runtime  to  control
       their behavior and query their settings.  Every daemon accepts a common
       set of commands documented under COMMON COMMANDS below.   Some  daemons
       support   additional   commands   documented  in  their  own  manpages.
       ovs-vswitchd in particular accepts a number of additional commands doc‐
       umented in ovs-vswitchd(8).

       The  ovs-appctl program provides a simple way to invoke these commands.
       The command to be sent is specified on  ovs-appctl's  command  line  as
       non-option arguments.  ovs-appctl sends the command and prints the dae‐
       mon's response on standard output.
```

as such, ovs-appctl is a sort of meta-command for controlling all the daemons. I think that access is fine for this interface, but I suspect that we'll need more rules once people use ovs-appctl in earnest. None of that is a blocker for this PR.

I would like to see an addition comment here:
```
# ovs-appctl allows sending commands to Open vSwitch daemons at runtime
/run/openvswitch/ovs-vswitchd.*.ctl rw,
/run/openvswitch/ovs-vswitchd.pid rw,
```